app/controllers/organizations_controller.rb

```diff
@@ -26,6 +27,10 @@ def create
   end
 
   def edit
+    unless is_owner
+        flash[:danger] = "User cannot edit this organization."
```

Comment:
What about "You are unauthorized to edit this organization"?

I think  it is more descriptive 😄

Reply:
@msarit sorry, I'm just seeing this now. Should I still make the tweak to the message?